src/utils/staking.js

```diff
@@ -1 +1,29 @@
 export const eraToDays = (era) => Math.round(era.toNumber() / 4);
+export const blockDuration = (startBlock, endBlock) => {
+  const blockEtaInSec = (endBlock - startBlock) * 6;
+  const days = Math.floor(blockEtaInSec / 86400);
+  const hours = Math.floor((blockEtaInSec - days * 86400) / 3600);
+  const minutes = Math.floor((blockEtaInSec - days * 86400 - hours * 3600) / 60);
+  const seconds = blockEtaInSec - days * 86400 - hours * 3600 - minutes * 60;
+
+  return {
+    days,
+    hours,
+    minutes,
+    seconds,
+  };
+};
+
+export const blockDurationFormatted = (startBlock, endBlock) => {
+  const {
+    days,
+    hours,
+    minutes,
+  } = blockDuration(startBlock, endBlock);
+
+  const formattedDays = days <= 0 ? '' : `${days} Days`;
+  const formattedHrs = hours <= 0 ? '' : `${hours} Hrs`;
+  const formattedMins = minutes <= 0 ? '' : `${minutes} Mins`;
+
+  return [formattedDays, formattedHrs, formattedMins].join(' ');
```

Comment:
If `endBlock - startBlock < 10` (a.k.a. days, hours & minutes are 0), this will return empty string. Instead, we should probably display something like `<1 min`.